### DISCUSSION TITLE
Foundry installation on WSL

### DISCUSSION THREAD

**Discussion by fiaz-tech** [Cyfrin/foundry-full-course-cu#2202] [Q&A]
After following the installation process, if I type foundryup this is what i get

foundryup: downloading latest forge, cast, anvil, and chisel
####################################################################################### 100.0%
foundryup: downloading manpages
####################################################################################### 100.0%
/home/omoluabi/.foundry/bin/forge: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /home/omoluabi/.foundry/bin/forge)
/home/omoluabi/.foundry/bin/forge: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.29' not found (required by /home/omoluabi/.foundry/bin/forge)
/home/omoluabi/.foundry/bin/forge: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /home/omoluabi/.foundry/bin/forge)
foundryup: command failed: /home/omoluabi/.foundry/bin/forge --version
foundryup: installed -

**Comment by EngrPips** [comment]
It seems like you haven't successfully installed `Foundry.` I will say uninstall whatever you have installed and restart the installation process. You can follow the installation process here https://book.getfoundry.sh/getting-started/installation

**Comment by elvisisvan** [comment]
this is likely a linux system (wsl) problem and not foundry, you need to update your dependencies before proceeding, run `sudo apt update` or similar commands specific to your installed distro

**Reply by fiaz-tech** [reply]
I just tried this and I'm still getting the 
same result. if i type " forge --version " this is what i get


forge: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by forge)
forge: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.29' not found (required by forge)
forge: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by forge)

**Reply by EngrPips** [reply]
Hey @fiaz-tech, look at this discussion https://github.com/Cyfrin/foundry-full-course-cu/discussions/2108 and see if it helps you solve this issue.
